components/style/values/specified/length.rs

```diff
@@ -205,12 +213,10 @@ impl CharacterWidth {
     }
 }
 
-/// A length.
-///
-/// https://drafts.csswg.org/css-values/#lengths
-#[derive(Clone, PartialEq, Debug)]
+/// A length that cannot parse `calc` expressions
```

Comment:
A `<length>`, without taking `calc` into account (and with spec links to both sections) would be the ideal IMO.